Sources/Models/EpisodeProgress.swift

```diff
@@ -0,0 +1,22 @@
+public struct EpisodeProgress: Codable, Equatable, Identifiable {
+  public let episodeSequence: Episode.Sequence
+  public let isFinished: Bool
+  public let percent: Int
+  public let userID: User.ID
+
+  public init(
+    episodeSequence: Episode.Sequence,
+    isFinished: Bool,
+    percent: Int,
+    userID: User.ID
+  ) {
+    self.episodeSequence = episodeSequence
+    self.isFinished = isFinished
+    self.percent = percent
+    self.userID = userID
+  }
+
+  public var id: Episode.Sequence {
```

Comment:
Dunno if we wanna tag this to the progress or not to avoid strange comparisons, or if that's too much to bother, ha. Maybe nested tagged values are also weird?

Reply:
Actually not sure why I did it this way. The `episode_progresses` table has a primary key, so I should just use that. Will update.